app/controllers/ReleasedController.scala

```diff
@@ -1,10 +1,12 @@
 package controllers
 
 import javax.inject.Inject
+import models.ReleasedMovieWithID
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5277123)